MH4U Dex/QuestContainerViewController.m

```diff
@@ -0,0 +1,61 @@
+//
+//  QuestContainerViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 4/13/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "QuestContainerViewController.h"
+
+#import "Quest.h"
+
+#import "QuestDetailsTableViewController.h"
+
+typedef NS_ENUM(NSInteger, QuestContainerSegmentedControlPage) {
+    Details = 0,
+    Rewards = 1,
+    
+    SegmentedControlPageCount
+};
+
+@interface QuestContainerViewController ()
+
+@property (nonatomic, weak) IBOutlet UIView *detailsSubView;
+@property (nonatomic, weak) IBOutlet UIView *rewardsSubView;
+
+@end
+
+@implementation QuestContainerViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    self.navigationItem.title = self.quest.name;
+    self.rewardsSubView.hidden = YES;
+}
+
+- (IBAction)segmentedControlChanged:(UISegmentedControl *)segmentedControl
+{
+    switch (segmentedControl.selectedSegmentIndex) {
+        case Details:
+            self.detailsSubView.hidden = NO;
+            self.rewardsSubView.hidden = YES;
+            break;
+        case Rewards:
+            self.detailsSubView.hidden = YES;
+            self.rewardsSubView.hidden = NO;
+            break;
+    }
+}
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
+{
+    if ([segue.identifier isEqualToString:@"embedQuestDetails"]) {
```

Comment:
You can get away with it here, but it's generally a good to put storyboard identifies in constants; structs are often used for this.